src/modules/about/AboutScreen.js

```diff
@@ -22,7 +22,7 @@ import {
 			"About": "About",
 			"Name": "Name",
 			"Version": "Version",
-			"Maintainer": "Maintainer",
+			"Vendor": "Maintainer",
```

Comment:
@t-pyrope I miss here the info about setting up the `Release` in the locales

Reply:
👍